tool/src/main/java/com/fidesmo/fdsm/CommandLineInterface.java

```diff
@@ -55,7 +56,8 @@ abstract class CommandLineInterface {
 
     final static protected OptionSpec<File> OPT_UPLOAD = parser.accepts("upload", "Upload CAP or recipe to Fidesmo").withRequiredArg().ofType(File.class).describedAs(".cap/.json file");
     final static protected OptionSpec<Void> OPT_LIST_APPLETS = parser.accepts("list-applets", "List applets at Fidesmo");
-    final static protected OptionSpec<String> OPT_DELETE_APPLET = parser.accepts("delete-applet", "Deletes applet at Fidesmo").withRequiredArg().describedAs("ID");
+    final static protected OptionSpec<String> OPT_DELETE = parser.acceptsAll(List.of("delete", "delete-applet"), "Deletes applets and recipes at Fidesmo").withRequiredArg().describedAs("ID");
```

Comment:
Not that it was heavily documented before, but maybe we could expand the description a bit stating that applet can be deleted by the LFDBH or cap file path? Unlikely anyone would be able to find this out by themselves or even remember how it works.